aiodocker/containers.py

```diff
@@ -190,6 +190,16 @@ def __init__(self, docker, **kwargs):
         await response.release()
         return
 
+    async def restart(self, **kwargs):
+        response = await self.docker._query(
+            "containers/{self._id}/restart".format(self=self),
+            method='POST',
+            headers={"content-type": "application/json"},
+            data=kwargs
```

Comment:
same here, we don't have data but a query param.

Reply:
I copied the implementation from the stop method because the docker docs state that the endpoints have exactly the same signature.

Maybe we should change this for the `stop` endpoint too?

I agree that we want to remove the header and use the timeout parameter explicitly.

Reply:
We can update the `stop` endpoint in another PR 👍, thanks for checking this.